software/sys_controller/ossc/menu.c

```diff
@@ -78,7 +78,7 @@ static void value_disp(alt_u8 v) { sniprintf(menu_row2, LCD_ROW_LEN+1, "    %u",
 static void lt_disp(alt_u8 v) { strncpy(menu_row2, lt_desc[v], LCD_ROW_LEN+1); }
 static void aud_db_disp(alt_u8 v) { sniprintf(menu_row2, LCD_ROW_LEN+1, "%d dB", ((alt_8)v-AUDIO_GAIN_0DB)); }
 static void vm_display_name (alt_u8 v) { strncpy(menu_row2, video_modes[v].name, LCD_ROW_LEN+1); }
-static void link_av_desc (avinput_t v) { strncpy(menu_row2, v == 0 ? "No link" : avinput_str[v], LCD_ROW_LEN+1); }
+static void link_av_desc (avinput_t v) { strncpy(menu_row2, v == AV_LAST ? "No link" : avinput_str[v], LCD_ROW_LEN+1); }
```

Comment:
You have to adapt [line 177](https://github.com/megari/ossc/blob/cab308fbd74104ea1d5abb4c1a9ae2c18954a328/software/sys_controller/ossc/menu.c#L177) to
{ LNG("Link prof->input","Link prof->input"), OPT_AVCONFIG_NUMVALUE,  { .num = { &tc.link_av,  OPT_WRAP, AV_RGBs, AV_LAST, link_av_desc } } },
with this commit, too

Reply:
Done.